femwell/culomb.py

```diff
@@ -57,9 +57,15 @@ def coulomb(u, v, w):
     )
 
     with tempfile.TemporaryDirectory() as tmpdirname:
-        mesh_from_OrderedDict(polygons, resolutions, filename=tmpdirname + '/mesh.msh', default_resolution_max=5)
+        mesh_from_OrderedDict(
+            polygons,
+            resolutions,
+            filename=f'{tmpdirname}/mesh.msh',
+            default_resolution_max=5,
+        )
+
```

Comment:
```suggestion
        mesh_from_OrderedDict(polygons, resolutions, filename=f'{tmpdirname}/mesh.msh', default_resolution_max=5)
```
The usage of f-strings is nice! I don't see the point in adding the line breaks here.